src/payment/strategies/digitalriver/digitalriver-payment-strategy.ts

```diff
@@ -31,60 +35,58 @@ export default class DigitalRiverPaymentStrategy implements PaymentStrategy {
 
     async initialize(options: PaymentInitializeOptions): Promise<InternalCheckoutSelectors> {
         this._digitalRiverInitializeOptions = options.digitalriver;
+        const paymentMethod = this._store.getState().paymentMethods.getPaymentMethodOrThrow(options.methodId);
 
-        const state = await this._store.dispatch(this._paymentMethodActionCreator.loadPaymentMethod(options.methodId));
-        const paymentMethod = state.paymentMethods.getPaymentMethodOrThrow(options.methodId);
-        let clientToken: DigitalRiverInitializeToken;
+        return this._digitalRiverScriptLoader.load()
+            .then((_window: DigitalRiverWindow) => {
```

Comment:
any reason why this was refactored to use `.then()` rather than `await` ? Using await is preferable as it's easier to read.

Other advantages:
https://mathiasbynens.be/notes/async-stack-traces